index.cjs

```diff
@@ -1,3 +1,6 @@
+// @ts-check
```

Comment:
This is somewhat atypical but this repo doesn't have a `tsconfig.json`, but I found that when I added it, the inference of the `import` and `json` plugins became `any`, instead of just inferring what the source the package returns is. As such, we don't have a proper `tsc` check. We'll just have to see if the editor reports any errors.